register-service-broker.sh

```diff
@@ -28,5 +28,25 @@ for SERVICE in $(echo "$SERVICES"); do
   # Must disable services prior to enabling, otherwise enable will fail if already exists
   # https://github.com/cloudfoundry/cli/issues/939
   cf disable-service-access "${ARGS[@]}"
-  cf enable-service-access "${ARGS[@]}"
```

Comment:
We should probably add a check like 

    if [ -n "${SERVICE_ORGANIZATION:-}" ] && [ -n "${SERVICE_ORGANIZATION_BLACKLIST:-}" ]; then
      echo "You may set SERVICE_ORGANIZATION or SERVICE_ORGANIZATION_BLACKLIST but not both"
      exit 1;
    fi

at the top of this script to warn if they try to specify conflicting params